pkgs/development/interpreters/perl/default.nix

```diff
@@ -1,11 +1,11 @@
 { config, lib, stdenv, fetchurl, pkgs, buildPackages, callPackage
-, enableThreading ? stdenv ? glibc, coreutils, makeWrapper
+, enableThreading ? true, coreutils, makeWrapper
```

Comment:
Wouldn't it make sense to change the default to `(stdenv.isDarwin or (stdenv ? glibc))`?

Reply:
I followed the text in the comment below, which says it defaults to true. Then I leave it to the assertion to pick it up. Realistically, perl is multithreaded on almost all platforms that Nix supports.

Reply:
The assertion makes it impossible to override, it's generally only done to ensure optional dependencies are available, etc.

Reply:
What do you think about removing the assertion altogether? Perl can be multithreaded on all platforms Nix supports, except in a special case of bootstrapping. Thus, I'd suggest defaulting to `true` on `enableThreading` and removing the assertion.

Reply:
I agree, the assert should be gone.